tests/vitest/math.test.ts

```
@@ -131,32 +180,373 @@ test.each([
+
+    const standard = math.pbc_dist(pos1 as math.Vec3, pos2 as math.Vec3, lattice)
+    const optimized = math.pbc_dist(
+      pos1 as math.Vec3,
+      pos2 as math.Vec3,
+      lattice,
+      lattice_inv,
+    )
+
+    expect(optimized).toBeCloseTo(standard, 10)
+    expect(optimized).toBeGreaterThanOrEqual(0)
+    expect(isFinite(optimized)).toBe(true)
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)